### PR TITLE
Added facebook feed to introtext

### PR DESCRIPTION
**Pull request by DieuwerH**
Added a Facebook feed next to the intro text, it jumps  under the text when resizing the screen

Implements #482

### CODE REVIEW

File: src/lancie-facebook/lancie-facebook.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+
+
+<dom-module id="lancie-facebook">
+  <template>
+    <style include="iron-flex iron-flex-alignment iron-flex-factors"></style>
```
Comment (TimvdLippe) [inline src/lancie-facebook/lancie-facebook.html]: You are not using these styles, so this include can be removed.

File: src/lancie-facebook/lancie-facebook.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
```
Comment (TimvdLippe) [inline src/lancie-facebook/lancie-facebook.html]: Unnecessary whitespace

File: src/lancie-facebook/lancie-facebook.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  <template>
+    <style include="iron-flex iron-flex-alignment iron-flex-factors"></style>
+    <style>
+      :host {
+        display: block;
+      }
+
+    </style>
+
+    <iframe src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Fareafiftylan%2F&tabs=timeline&width=500&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=false&appId" width="500" height="500" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
```
Comment (TimvdLippe) [inline src/lancie-facebook/lancie-facebook.html]: Put the attributes on new lines. The line is to wide atm

Comment (TimvdLippe) [reply src/lancie-facebook/lancie-facebook.html]: Move the inline styles to the style tag

Comment (TimvdLippe) [reply src/lancie-facebook/lancie-facebook.html]: The `&` show up red in the diff. Should they be escaped?

File: src/lancie-facebook/lancie-facebook.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+
+  </template>
+
+
```
Comment (TimvdLippe) [inline src/lancie-facebook/lancie-facebook.html]: Unnecessary whitespace

File: src/lancie-facebook/lancie-facebook.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <style>
+      :host {
+        display: block;
+      }
+
+    </style>
+
+    <iframe src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2Fareafiftylan%2F&tabs=timeline&width=500&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=false&appId" width="500" height="500" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
+
+
```
Comment (TimvdLippe) [inline src/lancie-facebook/lancie-facebook.html]: Unnecessary whitespace

File: src/lancie-home-page/lancie-home-page.html [hunk 6]
```diff
@@ -52,38 +53,66 @@
       width: 100%;
       display: block;
     }
+
+    lancie-facebook {
```
Comment (TimvdLippe) [inline src/lancie-home-page/lancie-home-page.html]: It is better to put this in the `:host` styling of the element

File: src/lancie-home-page/lancie-home-page.html [hunk 7]
```diff
@@ -52,38 +53,66 @@
...
+            also show off in this event, you can try your hand at laser gaming and you
+            can test your wit in a pop-up escape room. In the Launch Area, you'll find
+            pinball machines, airhockey tables, football tables and various retro gaming
+            consoles. An overview of things you can do besides gaming can be found
+            below.
+          </p>
+
+          <!-- <lancie-activities json="scripts/activity.json"></lancie-activities> -->
+        </div>
+
```
Comment (TimvdLippe) [inline src/lancie-home-page/lancie-home-page.html]: Unnecessary whitespace

**Comment by elarb** [comment]
I'm not seeing it for some reason. Also, if you want to use a [custom element](https://developers.google.com/web/fundamentals/web-components), you would be better of just extending `HTMLElement` since the element doesn't rely on things like binding or `dom-repeat`.

File: src/lancie-facebook/lancie-facebook.html (hunk 3 above)
Comment (Fastjur) [reply src/lancie-facebook/lancie-facebook.html]: I do not think the `&` should be escaped, that is just Github being funky, agree with 2 other statements.

File: bower.json [hunk 8]
```diff
@@ -6,12 +6,12 @@
     "web-animations-js": "web-animations/web-animations-js#^2.2.2",
     "google-map": "GoogleWebComponents/google-map#^2.0.0",
     "polymer": "polymer/polymer#^2.0.0",
-    "wisvch-footer": "wisvch/wisvch-footer#2.0-preview",
+    "wisvch-footer": "wisvch/wisvch-footer#2.0",
```
Comment (TimvdLippe) [inline bower.json]: I think you need to rebase, as the master version already points to `#^2.0.0`

**Comment by Fastjur** [comment]
I have tested this locally. It works fine!

Thanks for adding it!

File: src/lancie-home-page/lancie-home-page.html [hunk 9]
```diff
@@ -105,11 +125,11 @@
...
-          You can purchase your early bird ticket for €37,50* or a regular ticket at a later 
-          date for €40,-*. Both these tickets give access to everything named above. For an 
-          additional €2,50, we will pick up your PC and deliver it to the event. The maximum 
-          distance for the pickup service is 7.5km from Sports and Culture Unit of the TU 
-          Delft. Your PC will be returned to you on Sunday evening. Area FiftyLAN will be an  
+          You can purchase your early bird ticket for €37,50* or a regular ticket at a later
+          date for €40,-*. Both these tickets give access to everything named above. For an
+          additional €2,50, we will pick up your PC and deliver it to the event. The maximum
+          distance for the pickup service is 7.5km from Sports and Culture Unit of the TU
+          Delft. Your PC will be returned to you on Sunday evening. Area FiftyLAN will be an
```
Comment (Fastjur) [inline src/lancie-home-page/lancie-home-page.html]: What happened here?

File: src/lancie-home-page/lancie-home-page.html [hunk 10]
```diff
@@ -120,20 +140,20 @@
...
     <lancie-section type="secondary" header="Official Tournaments">
       <div>
-        <p>There are multiple official tournaments that will be organized by the committee. 
-          These tournaments feature large prize pools, so expect the competition to 
-          be fierce. If your favorite game is not listed here do not worry, you can organize 
-          your own! Contact us to get the ball rolling, more information on how to do that is 
+        <p>There are multiple official tournaments that will be organized by the committee.
+          These tournaments feature large prize pools, so expect the competition to
+          be fierce. If your favorite game is not listed here do not worry, you can organize
+          your own! Contact us to get the ball rolling, more information on how to do that is
```
Comment (Fastjur) [inline src/lancie-home-page/lancie-home-page.html]: And here?

File: src/lancie-home-page/lancie-home-page.html [hunk 11]
```diff
@@ -120,20 +140,20 @@
...
     </lancie-section>
 
     <lancie-section type="primary" header="Organize your own tournament!">
       <div>
-        <p>Is your favorite game not listed as an official tournament? We offer the 
-          possibility to host this tournament yourself. To set up your own tournament 
-          you need to write a proposal for your tournament. This proposal should contain the 
+        <p>Is your favorite game not listed as an official tournament? We offer the
+          possibility to host this tournament yourself. To set up your own tournament
+          you need to write a proposal for your tournament. This proposal should contain the
```
Comment (Fastjur) [inline src/lancie-home-page/lancie-home-page.html]: And here!

File: src/lancie-home-page/lancie-home-page.html [hunk 12]
```diff
@@ -145,17 +165,17 @@
...
-        <p>After we receive your proposal through the contact form, we will review it and hopefully accept it. 
-          When that happens, we appoint you as the responsible person to run the tournament. This 
-          means that entire organization of the tournament will be up to you. Our own support for 
-          the tournament will be minimal but we will help you gain other enthusiastic gamers by 
-          placing your tournament on our site for others to sign up. If you are interested in 
+        <p>After we receive your proposal through the contact form, we will review it and hopefully accept it.
+          When that happens, we appoint you as the responsible person to run the tournament. This
+          means that entire organization of the tournament will be up to you. Our own support for
+          the tournament will be minimal but we will help you gain other enthusiastic gamers by
+          placing your tournament on our site for others to sign up. If you are interested in
```
Comment (Fastjur) [inline src/lancie-home-page/lancie-home-page.html]: Here too!

**Comment by Fastjur** [comment]
@TimvdLippe Your requested changes have been implemented. Could you change your review?

**Comment by Fastjur** [comment]
I have, it looks good!
